.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: node_js
 
-# Opt-in to Travis's container infrastructure. This potentially makes builds less flaky, and easier to debug.
+# Force Travis CI to run jobs in VMs instead of Docker containers.
+# - https://blog.travis-ci.com/2018-10-04-combining-linux-infrastructures
```

Comment:
Should we remove the old blog links since they're deprecating that now?

Reply:
Done!